### PR TITLE
moving the docs to the terraform.io format

### PR DESCRIPTION
**Pull request by seansain**
Moving the docs to `website` directory and adding required changes for terraform.io docs to work.

Remaining docs task: Add docs for extraction_rule, partition, and content

After testing locally:
Screen Shot 2020-02-11 at 8 43 06 PM

### CODE REVIEW

File: website/docs/index.html.markdown [hunk 1]
```diff
@@ -68,10 +75,5 @@ $ terraform plan
 ## Argument Reference
 - `access_id` - (Optional) This is the Sumo Logic Access ID. It must be provided, but it can also be source from the SUMOLOGIC_ACCESSID environment variable.
```
Comment (sumovishal) [inline website/docs/index.html.markdown]: pre-existing: `access_id` is not optional.

File: website/docs/index.html.markdown [hunk 2]
```diff
@@ -68,10 +75,5 @@ $ terraform plan
...
 - `access_key` - (Optional) This is the Sumo Logic Access Key. It must be provided, but it can also be sourced from the SUMOLOGIC_ACCESSKEY variable.
```
Comment (sumovishal) [inline website/docs/index.html.markdown]: pre-existing: `access_key` is not optional.

File: website/docs/index.html.markdown [hunk 3]
```diff
@@ -68,10 +75,5 @@ $ terraform plan
...
-- `environment` - (Optional) This is the API endpoint to use. Default is `us2`. See the [Sumo Logic documentation][1] for details on which environment you should use. This value can also be source from the SUMOLOGIC_ENVIRONMENT variable.
+- `environment` - (Optional) This is the API endpoint to use. Default is `us2`. See the [Sumo Logic documentation](https://help.sumologic.com/APIs/General_API_Information/Sumo_Logic_Endpoints_and_Firewall_Security) for details on which environment you should use. This value can also be source from the SUMOLOGIC_ENVIRONMENT variable.
```
Comment (sumovishal) [inline website/docs/index.html.markdown]: I think we should remove default value for `environment` var and make it a required field.

File: website/docs/index.html.markdown [hunk 4]
```diff
@@ -68,10 +75,5 @@ $ terraform plan
...
 - `base_url` - (Optional) This is the API endpoint base URL to use. You can specify that instead of the environment. It can also be sourced from SUMOLOGIC_BASE_URL environment variable.
```
Comment (sumovishal) [inline website/docs/index.html.markdown]: I don't think this var needs to be documented publicly. `environment` should be enough.

**Comment by rvmiller89** [comment]
Looking at the screenshot, I noticed one issue: our name is wrong!

It's actually `Sumo Logic`, and not `Sumologic`.

**Pull request merged**